packages/create-block/lib/templates/es5/readme.txt.mustache

```diff
@@ -0,0 +1,54 @@
+=== {{title}} ===
+Contributors:      {{author}}
+Tags:              block
+Requires at least: 5.3.2
+Tested up to:      5.3.2
+Stable tag:        {{version}}
+Requires PHP:      7.0.0
+License:           {{license}}
+License URI:       {{{licenseURI}}}
+
+Here is a short description of the plugin. This should be no more than 150 characters. No markup here.
```

Comment:
How about we put `{{description}}` instead?

```suggestion
{{description}}
```

and copy the hints on how to add the long description in the next section?

Reply:
I meant the one from the template shared by @ntwb:

https://wordpress.org/plugins/readme.txt